src/bumerang/app.py

```diff
@@ -35,6 +37,9 @@ def __init__(self):
             (r'/profile/([0-9]+)/?', ProfileHandler),
             (r'/request/?', BorrowHandler),
             (r'/request/([0-9]+)/?', BorrowHandler),
+            (r'/requests/recent/?', BorrowsByRecentHandler),
```

Comment:
This was giving me issues... I could combine them into the same regex, ie /requests/recent/?([0-9]+)?/?, or something like that, but I'm unable to use the default "num_requests" parameter when the uri is "/requests/recent", instead it defaults to None.

So it depends what we think is cleaner; having a check in the BorrowsByRecentHandler class to see if num_requests is None, then set it to the default value, or just having two URIs and using the default parameter value in the get method